cockatrice/src/gameselector.cpp

```diff
@@ -212,3 +216,17 @@ void GameSelector::processGameInfo(const ServerInfo_Game &info)
 {
     gameListModel->updateGameList(info);
 }
+
+void GameSelector::updateButtonChoices(const QModelIndex &index)
+{
+    if (!index.isValid())
+        return;
+
+    const ServerInfo_Game &game = gameListModel->getGame(index.data(Qt::UserRole).toInt());
+    bool overrideRestrictions = !tabSupervisor->getAdminLocked();
```

Comment:
Imho having a variable named `overrideRestrictions` helps understanding what is actually checked in the `if()`, while just putting `!tabSupervisor->getAdminLocked()` would make it more obscure, as you originally noticed by not getting why the check was performed.